scripts/static-tests

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Builds statically compiled test binaries. Useful for cross-compiling.
+#
+# The build produces the file "static-tests.tar.gz" which is meant to be
+# untarred on the target. After untarring, run ./scripts/static-tests to
+# start the extracted tests.
+
+# Some options that were useful for arm64:
+# GOARCH=arm64 ETCD_UNSUPPORTED_ARCH=arm64 ./scripts/static-tests
+
+set -eo pipefail
+
+OUTDIR="static-tests"
+
+if [ -e "$OUTDIR" ]; then
+	echo "found "$OUTDIR"/ running tests..."
```

Comment:
when executing the tests, it seems `default.etcd/` is created, so you should trap rm -rf default.etcd here i think.